feat(changelog): data-warehouse-added-data-warehouse-for-clickhouser-now 2026-08-07 (#6763)
scaleway/docs-content: changelog/august2026/2026-08-07-data-warehouse-added-data-warehouse-for-clickhouser-now.mdx

---
title: Data Warehouse for ClickHouse® now offers SLA
status: added
date: 2026-08-07
category: data-and-analytics
product: data-warehouse
---

> We now offer SLA for Data Warehouse for ClickHouse®, guaranteeing high availability and reliability for your production analytical workloads. Refer to the [Data Warehouse for ClickHouse® Service Level Agreement](https://www.scaleway.com/en/data-warehouse-for-clickhouser/sla/) for additional details.

Reviewer: ```suggestion
Data Warehouse for ClickHouse® now offers a Service Level Agreement (SLA), guaranteeing high availability and reliability for your production analytical workloads. See the [Data Warehouse for ClickHouse® Service Level Agreement](https://www.scaleway.com/en/data-warehouse-for-clickhouser/sla/) for details.
```
